### ISSUE TITLE
Low protected SOM pools

### ISSUE THREAD

**Issue by wwieder** [wwieder/biogeochem_testbed_1.1#47]
Work that Kat's done suggests the protected pools in MIMICS are too small, both in absolute amounts of SOMp and the fraction of total C that's physically protected (MAOM / SOMp).  This may be especially true in the Arctic.

Towards this end, I've tried: 

1. **Increasing the fraction of microbial turnover that is physically protected (MIC -> SOMp)**
    - Especially in low clay soils by increasing the intercept and decreasing the slope for `fPHYS_r` and `fPHYS_k`. New parameterizations is in blue.
![image](https://user-images.githubusercontent.com/8031012/113887204-88b26c00-977e-11eb-927f-7b2173186154.png)
    - This increases SOMp pools globally, but mostly in the tropics where we may already have a high bias in total SOC stocls

2. **Increasing decomposition rates, especially in colder systems.**
    - This was motivated by the very large LIT stocks that were accumulating in high latitudes (4-5x the size of SOMp pools), suggesting that decomposition rates are too slow.  (See also lower than observed microbial biomass at high latitudes.)
    - This was done by decreasing Km in the model, specifically decreasing the intercept `Kint` and increasing the slope `Kslope` of the temperature function used, new parameterizations is in blue.
![image](https://user-images.githubusercontent.com/8031012/113886491-f27e4600-977d-11eb-9e80-ff7f260b9f6e.png)
    - This may need further modification, or focus on Vmax instead?

3. **Adding a temperature sensitivity to the desorption rate controlling the transfer of SOMp -> SOMa, see #46.**
    - Previously this was simply a function of soil clay content.
    - The new function adds a Q10 of 1.3 to this flux with a reference temperature of 25 C.
    - The plot below shows desorption rates for a soil with 30% clay as a function of temperature for the orig. parameteriztion (black) and the new one (blue)
![image](https://user-images.githubusercontent.com/8031012/113888614-bb109900-977f-11eb-9170-3ab6d85d3c13.png)
    - The q10 of this flux may need to be decreased, as the change in the flux is pretty large!
    - It will be interesting to see how this changes the temperature sensitivity of the model and seasonal cycle of CO2 fluxes.

**Comment by wwieder** [comment]
preliminary results are below, but we have two parameterizations that favor greater SOC storage in `LIT` and `SOMc` pools (default parameterization reflected in # 1 and # 2),  vs. `SOMp` pools in temperature sensitive desorption case, # 4.   
![image](https://user-images.githubusercontent.com/8031012/114577941-b180a880-9c39-11eb-98ea-c8d09092fa1c.png)

**Comment by piersond** [comment]
Is it possible for me to get a version of the forcing data you're using here that would work with the sandbox? It would be super fun to run it through the MC parameterization script, then take some of those parameter combinations, and run them back through the time-sensitive version of MIMICS.

Kat and I discussed that she might have a global gridded dataset, but I'm wondering if it may be more useful to have a dataset that directly aligns with what you're showing here.

If it's just a matter of crunching 30 min interval data into annual means, send it my way and I'll figure it out.

**Comment by wwieder** [comment]
yep, easiest is to get you onto the ncar machines and I can point you to where this all lives.
do you still have your login, password and DUO authentications working?

**Comment by piersond** [comment]
Cool! Unfortunately I can't seem to get myself logged into NCAR anymore.

**Comment by wwieder** [comment]
You can try writing to CGD Support ***@***.***> or CISL Support <
***@***.***>.  The machines you need to get onto are with CGD, but
CISL may be the one that actually sets your access to NCAR computing.

**Comment by piersond** [comment]
Thanks! The email addresses didn't come through. I'll give it a Google and move this conversation to an email.
